src/sentry/static/sentry/app/components/betaTag.tsx

```diff
@@ -7,13 +7,15 @@ import space from 'app/styles/space';
 import {t} from 'app/locale';
 
 type Props = {
+  hasTooltip?: boolean;
```

Comment:
Is this explicit prop necessary? `title` being falsey implies no tooltip

Reply:
are you suggesting `title?: string | boolean;`? with that, I suppose you could pass in `title={true}` and get an empty tooltip. also, the Tooltip title prop is a `React.ReactNode`

Reply:
I'm more suggesting that `Tooltip.disabled` can be determined by casting `title` as a boolean so if you wanted to disable tooltips you can do `<BetaTag title={null} />`

Reply:
I agree w/ @billyvg